pkg-manager/core/test/install/peerDependencies.ts

```diff
@@ -1324,3 +1324,45 @@ test('deduplicate packages that have peers, when adding new dependency in a work
   expect(depPaths).toContain(`/@pnpm.e2e/abc/1.0.0${createPeersFolderSuffix([{ name: '@pnpm.e2e/peer-a', version: '1.0.0' }, { name: '@pnpm.e2e/peer-b', version: '1.0.0' }, { name: '@pnpm.e2e/peer-c', version: '1.0.0' }])}`)
   expect(depPaths).toContain(`/@pnpm.e2e/abc-parent-with-ab/1.0.0${createPeersFolderSuffix([{ name: '@pnpm.e2e/peer-c', version: '1.0.0' }])}`)
 })
+
+test('find correct peer dependency with npm aliases', async () => {
```

Comment:
It is just aliases, not "npm aliases"
```suggestion
test('find correct peer dependency with aliases', async () => {
```